### PR TITLE
[New] Detonate LNK TOP Rules

### PR DESCRIPTION
**Pull request by Samirbous**
the following two Elastic Defend endpoint rules are the top ones matching on TPs and SIEM FNs from detonate for LNK files, converting them to SIEM rules compatible with Sysmon/Winlogbeat, SentinelOne and M365 Defender datasources :

https://github.com/elastic/protections-artifacts/blob/ea2f8dd3b61a7cdf2ce83ca5f06f2096bb62a494/behavior/rules/windows/execution_suspicious_powershell_execution.toml#L8

https://github.com/elastic/protections-artifacts/blob/ea2f8dd3b61a7cdf2ce83ca5f06f2096bb62a494/behavior/rules/windows/execution_suspicious_windows_command_shell_execution.toml#L8

NB: I've omitted Elastic Defend and Endgame as datasources to avoid alert duplicates (if a user has one of those integration enabled it won't bring value to have SIEM and Endpoint alert for the same thing).

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: New - Guidelines

These guidelines serve as a reminder set of considerations when proposing a new rule.

### Documentation and Context

- [ ] Detailed description of the rule.
- [ ] List any new fields required in ECS/data sources.
- [ ] Link related issues or PRs.
- [ ] Include references.

### Rule Metadata Checks

- [ ] `creation_date` matches the date of creation PR initially merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive, considering performance for diverse environments. Non ecs fields should be added to `non-ecs-schema.json` if not available in an integration.
- [ ] `min_stack_comments` and `min_stack_version` should be included if the rule is only compatible starting from a specific stack version.
- [ ] `index` pattern should be neither too specific nor too vague, ensuring it accurately matches the relevant data stream (e.g., use logs-endpoint.process-* for process data).
- [ ] `integration` should align with the `index`. If the integration is newly introduced, ensure the manifest, schemas, and `new_rule.yaml` template are updated.
- [ ] `setup` should include the necessary steps to configure the integration.
- [ ] `note` should include any additional information (e.g. Triage and analysis investigation guides, timeline templates).
- [ ] `tags` should be relevant to the threat and align/added to the `EXPECTED_RULE_TAGS` in the definitions.py file.
- [ ] `threat`, `techniques`, and `subtechniques` should map to ATT&CK always if possible.

#### New BBR Rules
- [ ] `building_block_type` should be included if the rule is a building block and the rule should be located in the `rules_building_block` folder.
- [ ] `bypass_bbr_timing` should be included if adding custom lookback timing to the rule.

### Testing and Validation

- [ ] Provide evidence of testing and detecting the expected threat.
- [ ] Check for existence of coverage to prevent duplication.

**Review by Aegrah — APPROVED** [review]

**Review by w0rk3r — APPROVED** [review]
We can make these fields optional (`?field`), but if these exclusions are noisy, you may want to use different fields to filter them out.

File: rules/windows/execution_windows_cmd_shell_susp_args.toml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                 "?:\\Program Files (x86)\\Tier2Tickets\\button_gui.exe",
+                 "?:\\Program Files (x86)\\Helpdesk Button\\button_gui.exe",
+                 "?:\\VTSPortable\\VTS\\jre\\bin\\javaw.exe",
+                 "?:\\Program Files\\Bot Framework Composer\\Bot Framework Composer.exe",
+                 "?:\\Program Files\\KMSYS Worldwide\\eQuate\\*\\SessionMgr.exe",
+                 "?:\\Program Files (x86)\\Craneware\\Pricing Analyzer\\Craneware.Pricing.Shell.exe",
+                 "?:\\Program Files (x86)\\jumpcloud-agent-app\\jumpcloud-agent-app.exe",
+                 "?:\\Program Files\\PostgreSQL\\*\\bin\\pg_dumpall.exe",
+                 "?:\\Program Files (x86)\\Vim\\vim*\\vimrun.exe") and
+  not process.working_directory : "?:\\Program Files (x86)\\Spiceworks\\" and
```
[the remark this answers is not in the document]
Comment (Samirbous) [reply rules/windows/execution_windows_cmd_shell_susp_args.toml]: removed exclusion in favour of compatibility

File: rules/windows/execution_windows_cmd_shell_susp_args.toml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  not process.command_line :
+               ("\"cmd\" /c %NETBEANS_MAVEN_COMMAND_LINE%",
+                "?:\\Windows\\system32\\cmd.exe /q /d /s /c \"npm.cmd ^\"install^\" ^\"--no-bin-links^\" ^\"--production^\"\"") and
+  not (process.name : "cmd.exe" and process.args : "%TEMP%\\Spiceworks\\*" and process.args : "http*/dataloader/persist_netstat_data") and
+  not process.parent.command_line : "cmd  /q /k  prompt MAM:Remote$G" and
+  not process.parent.args :  "C:\\WINDOWS\\Runtime\\utility\\wrapper.vbs" and
+  not (process.parent.name : "wscript.exe" and
+       process.parent.args : ("C:\\IBM\\ITM\\TMAITM6_x64\\K06_uninstall.vbs", "C:\\Windows\\System32\\gatherNetworkInfo.vbs")) and
+  not (process.args == "echo" and process.args == "GEQ" and process.args == "1073741824") and
+  not (process.parent.name : "wscript.exe" and process.parent.args : "C:\\Program Files (x86)\\CaseWare\\Template\\FinancialsIFRS\\Packager\\After\\0\\FinancialsIFRSCreateShortcuts.js")
```
[the remark this answers is not in the document]
Comment (Samirbous) [reply rules/windows/execution_windows_cmd_shell_susp_args.toml]: removed exclusion in favour of compatibility

File: rules/windows/execution_windows_cmd_shell_susp_args.toml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  not (process.args :  "?:\\Program Files\\Citrix\\Secure Access Client\\nsauto.exe" and process.parent.name : "userinit.exe") and
+  not process.args :
+            ("?:\\Program Files (x86)\\PCMatic\\PCPitstopScheduleService.exe",
+             "?:\\Program Files (x86)\\AllesTechnologyAgent\\*",
+             "https://auth.axis.com/oauth2/oauth-authorize*") and
+  not process.command_line :
+               ("\"cmd\" /c %NETBEANS_MAVEN_COMMAND_LINE%",
+                "?:\\Windows\\system32\\cmd.exe /q /d /s /c \"npm.cmd ^\"install^\" ^\"--no-bin-links^\" ^\"--production^\"\"") and
+  not (process.name : "cmd.exe" and process.args : "%TEMP%\\Spiceworks\\*" and process.args : "http*/dataloader/persist_netstat_data") and
+  not process.parent.command_line : "cmd  /q /k  prompt MAM:Remote$G" and
```
[the remark this answers is not in the document]
Comment (Samirbous) [reply rules/windows/execution_windows_cmd_shell_susp_args.toml]: removed exclusion in favour of compatibility

File: rules/windows/execution_windows_cmd_shell_susp_args.toml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+ (
+
+  process.command_line : ("*).Run(*", "*GetObject*", "* curl*regsvr32*", "*echo*wscript*", "*echo*ZONE.identifier*",
+  "*ActiveXObject*", "*dir /s /b *echo*", "*unescape(*",  "*findstr*TVNDRgAAAA*", "*findstr*passw*", "*start*\\\\*\\DavWWWRoot\\*",
+  "* explorer*%CD%*", "*%cd%\\*.js*", "*attrib*%CD%*", "*/?cMD<*", "*/AutoIt3ExecuteScript*..*", "*&cls&cls&cls&cls&cls&*",
+  "*&#*;&#*;&#*;&#*;*", "* &&s^eT*", "*& ChrW(*", "*&explorer /root*", "*start __ & __\\*", "*findstr /V /L *forfiles*",
+  "*=wscri& set *", "*http*!COmpUternaME!*", "*start *.pdf * start /min cmd.exe /c *\\\\*", "*pip install*System.Net.WebClient*",
+  "*Invoke-WebReques*Start-Process*", "*-command (Invoke-webrequest*", "*copy /b *\\\\* ping *-n*", "*echo*.ToCharArray*") or
+
+  (process.args_count == 3 and process.args : "%*%" and process.args:"/c") or
```
[the remark this answers is not in the document]
Comment (Samirbous) [reply rules/windows/execution_windows_cmd_shell_susp_args.toml]: removed exclusion in favour of compatibility

**Comment by Samirbous** [comment]
> We can make these fields optional (`?field`), but if these exclusions are noisy, you may want to use different fields to filter them out.

removed exclusions in favour of compatibility

**Pull request merged**